test/index.js

```diff
@@ -1,8 +1,36 @@
-const expect = require('chai').expect,
+const expect = require('chai').expect;
 const toRGBAArray = require('..');
+const Canvas = require('canvas');
+const range = require('lodash.range');
+const fs = require('fs');
+const path = require('path');
 
-describe('my thing', function () {
-  it('should work', function () {
-    toRGBAArray('thing');
+describe('to-rgba-array', function () {
+  it('will work with a canvas', function () {
+    var canvas = new Canvas(200, 200);
+    return toRGBAArray(canvas).then(function (result) {
+      expect(result).to.be.a('Uint8ClampedArray');
```

Comment:
Here's some code from one of my modules. I use the expected output in a fixture file:

https://github.com/eugeneware/gifencoder/blob/master/test/index.js#L139-L148

With RGBA data it can get quite large. So perhaps use a smallish file like a 128x128 pixel image with some alpha and do a full comparison.